roles/openshift_examples/files/examples/v1.2/xpaas-streams/jboss-image-streams.json

```diff
@@ -152,8 +152,8 @@
                             "version": "1.3"
                         },
                         "from": {
-                          "Kind": "DockerImage",
-                          "Name": "registry.access.redhat.com/jboss-eap-7-beta/eap70-openshift:1.3"
+                          "kind": "DockerImage",
+                          "name": "registry.access.redhat.com/jboss-eap-7-beta/eap70-openshift:1.3"
                         }
```

Comment:
crap, yeah. filed an issue with upstream of us so we don't inadvertantly re-introduce this https://github.com/jboss-openshift/application-templates/issues/144